### DISCUSSION TITLE
Build in delay before cross-seed runs after qBT downloads?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#497] [Q&A] [answered]
Is there a way to build in a delay before cross-seed runs after a download completes in qBT in order to give Sonarr/Radarr/etc time to do it's thing before it tries to inject cross-seeded torrents?

**Comment by zakkarry** [comment, marked as answer]
I provided you a script on discord thread for this, for anyone seeking to replicate this until, perhaps, custom categories is added, execute the following bash script.

You'll want to run something like `sh /path/to/script.sh %I` and replace the localhost with your applicable cross-seed daemon address.

#!/bin/bash
sleep 60
curl -XPOST http://localhost:2468/api/webhook --data-urlencode "infoHash=$1"

**Reply by SawkeeReemo** [reply]
Works perfectly. Thank you!

**Reply by SawkeeReemo** [reply]
Follow up question: Does this stack? Like, if it runs once, but then gets triggered again before the 60 seconds is up, will those queue or stack up in any way?

**Reply by zakkarry** [reply]
It will trigger like any completion script, and run 60s from when it was executed. If you run 3 within 10s, they will execute in that order, each 60s from when they were triggered by qbit.

**Reply by SawkeeReemo** [reply]
Awesome. Thanks! I thought so, but wanted to make sure. Appreciate the info as always!
